src/pages/home/ReportScreen.js

```diff
@@ -191,6 +191,7 @@ class ReportScreen extends React.Component {
                                 reportID={reportID}
                                 reportActions={this.props.reportActions}
                                 report={this.props.report}
+                                isFocused={_.size(this.props.reportActions) === 1}
```

Comment:
Do we really need this prop? If I am not missing something `reportActions={this.props.reportActions}` should be enough and `_.size(this.props.reportActions) === 1` can be moved inside the component.

Reply:
the component is already expecting the isFocused prop that's why i used it only

Reply:
@parasharrajat

Reply:
Could you please explain this to me? what do you mean by expecting it?

Reply:
/** Is composer screen focused */
    isFocused: PropTypes.bool.isRequired,

already as PROP

Reply:
and using it 
![image](https://user-images.githubusercontent.com/31244700/135979929-0032b3a9-ff74-4186-95dd-69e341ba6221.png)
and in did update in ReportActionCompose.js

Reply:
This is coming from ` withNavigationFocus` thus you should not depend on this.

Reply:
ok, let me use that as you suggested.

Reply:
The keyboard never opens for the new chat on Android.